src/main/java/net/minecraftforge/fluids/Fluid.java

```diff
@@ -299,9 +305,18 @@ public final int getViscosity()
         return this.viscosity;
     }
 
+    /**
+     * @deprecated Use isLighterThanAir() instead
+     */
+    @Deprecated
     public final boolean isGaseous()
```

Comment:
Can we retain the old method since its name is shorter?

Reply:
I'm still for keeping the gas flag and the lighter-than-air property apart. The change to use isLighterThanAir() for all rendering code is a good one, but the gas property could be useful for mods. Maybe add a comment to the javadoc along the lines of "mods should not allow fluids that are gasses into non-airtight containers", but otherwise do nothing with it in Forge.

Maybe we could even get Mek to drop its gas API and use Forge gasses instead then. ;)

Reply:
"mods should not allow fluids that are gasses into non-airtight containers"
This is exactly the point we (Mekanism) use a separate gas api. If this becomes part of Forge and widely used, I will personally make sure Mekanism gets a gas rewrite :)

Reply:
Yes, we should actually separate these two. Some gas can be heavier than air while flowing in a very weird manner.

Reply:
Yep - this was the initial intention - the Fluid API is for liquids and gasses, and it's always been a bit strange that Mek felt a need to roll their own.

Reply:
Strange? I don't think it was strange, as that gas flag was used for rendering instead of being a pure property like e.g. temperature.

Reply:
i think it would make sense to replace it with "needsAirtightContainer" or something to that extent - that way it's self-documenting. Makes it easier to get everyone on board